pkg/sensors/manager.go

```diff
@@ -344,13 +356,15 @@ type Manager struct {
 type tracingPolicyAdd struct {
 	ctx     context.Context
 	name    string
+	ck      collectionKey
```

Comment:
Do we still need the name here? Isn't it `ck.name`?

Reply:
Deleted `name` in the struct